pkg/util/mount/fake.go

```diff
@@ -175,7 +179,35 @@ func (f *FakeMounter) PathIsDevice(pathname string) (bool, error) {
 }
 
 func (f *FakeMounter) GetDeviceNameFromMount(mountPath, pluginDir string) (string, error) {
-	return getDeviceNameFromMount(f, mountPath, pluginDir)
+	if runtime.GOOS == "darwin" {
+		return GetDeviceNameFromMountDarwin(f, mountPath, pluginDir)
+	} else {
+		return getDeviceNameFromMount(f, mountPath, pluginDir)
+	}
+}
+
+// getDeviceNameFromMount find the device name from /proc/mounts in which
```

Comment:
This comment seems to be written specifically for Linux, need we add a more related comment for this new method?